delius-auto-test/sub-projects/delius-core.tfvars

```diff
@@ -0,0 +1,48 @@
+# delius-core-sandpit  delius-core.tfvar
+## Delius Core Specific
+
+egress_443 = true
+egress_80 = true
+
+ndelius_version = "4.1.8.4"
+
+# ref ../../common/common.tfvars
+db_size_delius_core = {
+  database_size  = "small"
+  instance_type  = "t3.large"
+  disk_iops      = 1000
+  disks_quantity = 2  # Do not decrease this
+  disk_size      = 200 # Do not decrease this
+  # total_storage  = 400 # This should equal disks_quantity x disk_size
+}
+
+ansible_vars_oracle_db = {
+  service_user_name             = "oracle"
+  database_global_database_name = "DATNDA"
+  database_sid                  = "DATNDA"
+  ## oradb_sys_password            = "/${environment_name}/delius-core/oracle-database/db/oradb_sys_password"
+  ## oradb_system_password         = "/${environment_name}/delius-core/oracle-database/db/oradb_system_password"
+  ## oradb_sysman_password         = "/${environment_name}/delius-core/oracle-database/db/oradb_sysman_password"
+  ## oradb_dbsnmp_password         = "/${environment_name}/delius-core/oracle-database/db/oradb_dbsnmp_password"
+  ## oradb_asmsnmp_password        = "/${environment_name}/delius-core/oracle-database/db/oradb_asmsnmp_password"
+  database_characterset         = "AL32UTF8"
+  database_bootstrap_restore    = "True" # whether primary db has db restore on bootstrap
+  database_backup               = "dbbackup/dev/delius" # path in S3 to directory backup files
+  database_backup_sys_passwd    = "/dbbackup/delius-core-dev/delius-core/oracle-database/db/oradb_sys_password" # ssm parameter store name for db backup password
```

Comment:
does this need to be env-specific instead of delius-core-dev?

Reply:
Not yet, as we only have the one gold back up. But soon it will need a specific path and version.
Well spotted - thanks